actions/version_number/action.yaml

```diff
@@ -72,6 +75,7 @@ runs:
         REF_PREFIX="refs/heads/"
         BRANCH_NAME=${BRANCH_NAME_WITH_REF#$REF_PREFIX}
         BRANCH_NAME_TRUNC=${BRANCH_NAME#50}
+        echo "simple_version=$NBGV_SimpleVersion" >> $GITHUB_OUTPUT
```

Comment:
I recognised that on line 85 we can pass in `inputs.short_name` to return the simple version of `new_version`.  As I'm running this workflow on the feature branch, I can't bypass line83 so I have to add a new output. 

Do you think it's okay to keep this `simple_version` output or should I just pass in the `inputs.short_name`? @thomasm-ttd